staubli_tx2-90_support/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(staubli_tx2-90_support)
```

Comment:
Thanks for reviewing! This is actually following the official Stäubli naming scheme. Besides that, dropping the dash could also lead to some confusion as they already have a TX200 and a TX340 in their portfolio and might or might not add others.

Reply:
But then, for consistency, shouldn't we rename all existing ones as well, eg `tx60` to `tx-60`, and so on?

Reply:
Perhaps @gavanderhoorn has a more defined opinion on this matter.

Reply:
Dashes are not legal in ROS resource names, so those will have to go. See also [wiki/Names](http://wiki.ros.org/Names) and [REP-I7: Naming conventions for ABB robots in ROS-Industrial](https://github.com/ros-industrial/rep/blob/bbd88b9b1dc9b61be5178c8b3a93b9bec488702a/rep-I0007.rst).

My first proposal would be to convert it to an underscore: `staubli_tx2_90_support`.

I'm not too thrilled about it, but given our options I believe this makes the most sense: easily readable, we don't overlap with other existing series numbers, etc.

Reply:
👍   @gavanderhoorn what should we do then with the existing packages? If a breaking change is acceptable, I can send a new PR with all the renaming. 
(fwiw: considering this is the `experimental` repo, I would strongly advocate for a breaking change in the name of consistency, also because the impact of breakage is very small)

Reply:
> But then, for consistency, shouldn't we rename all existing ones as well, eg `tx60` to `tx-60`, and so on?

Their have the same inconsistency in their naming, it's TX60 and TX2-60 on their side

Reply:
> Their have the same inconsistency in their naming, it's TX60 and TX2-60 on their side

Yes, but we cannot fix that, we can however, make it internally consistent within the ROS-I ecosystem.

Reply:
I'm not sure I understand.

Aren't all TX2 series robots named TX2-something_something? And all non-TX2 series: TXsomething_something?

Reply:
> Dashes are not legal in ROS resource names, so those will have to go.

Good point

> My first proposal would be to convert it to an underscore: `staubli_tx2_90_support`.
I'm fine with that as well, I can change the TX2 packages.

I would however rather not change the TX packages needlessly in order not to cause troubles to existing users

Reply:
Yes, those are the first TX2 packages.

Reply:
Ah, now I understand. @gonzalocasas suggested to also do this: `staubli_tx_60_support`.

No, that would not be needed and I feel also undesirable. The underscores are only used when really needed.

See also again `REP-I7`.

Reply:
Perfect!